controllers/clients.js

```diff
@@ -7,28 +7,23 @@ const listClients = (req, res) => {
 };
 
 const createClient = (req, res) => {
-  const client = new Clients({
+  const clientCreated = new Clients({
     name: req.body.name,
     phone: req.body.phone,
-    location: {
-      country: req.body.location.country,
-      state: req.body.location.state,
-      city: req.body.location.city,
-      address: req.body.location.address,
-    },
+    location: req.body.location,
     logo: req.body.logo,
     description: req.body.description,
     created: {
       admin: req.body.created.admin,
-      timestamp: new Date().getTime().toString(),
+      timestamp: new Date().toISOString(),
     },
     modified: {
       admin: req.body.modified.admin,
-      timestamp: new Date().getTime().toString(),
+      timestamp: new Date().toISOString(),
```

Comment:
it is not necessary since we are using timestamp on schema

Reply:
Removed